entrypoint.sh

```diff
@@ -1,23 +1,3 @@
 #!/usr/bin/env bash
 
-{
-
-if [ ! -z "$EXPOSE_SSH" ]; then
-  echo "Starting SSH server"
-  /usr/sbin/sshd -e
-fi
-
-ldconfig
```

Comment:
note this change: I don't think it's necessary as we run `RUN ldconfig` during image build